### PR TITLE
fix :  Undefined array key rowid in C:\wamp64\www\dolibarr-dev\htdocs…

### PR DESCRIPTION
**Pull request by grandoc**
…\core\class\html.formfile.class.php on line 1392

### CODE REVIEW

File: htdocs/core/class/html.formfile.class.php [hunk 1]
```diff
@@ -1290,7 +1290,7 @@ public function list_of_documents($filearray, $object, $modulepart, $param = '',
 				if ($file['name'] != '.'
 						&& $file['name'] != '..'
 						&& !preg_match('/\.meta$/i', $file['name'])) {
-					if ($filearray[$key]['rowid'] > 0) {
+					if (!empty($filearray[$key]['rowid'])) {
```
Comment (eldy) [inline htdocs/core/class/html.formfile.class.php:1293]: must use
if (!empty($filearray[$key]['rowid'] && $filearray[$key]['rowid'] > 0) {

to have same effect.

File: htdocs/core/class/html.formfile.class.php [hunk 2]
```diff
@@ -1300,7 +1300,7 @@ public function list_of_documents($filearray, $object, $modulepart, $param = '',
 					print '<!-- Line list_of_documents '.$key.' relativepath = '.$relativepath.' -->'."\n";
 					// Do we have entry into database ?
 					print '<!-- In database: position='.$filearray[$key]['position'].' -->'."\n";
-					print '<tr class="oddeven" id="row-'.($filearray[$key]['rowid'] > 0 ? $filearray[$key]['rowid'] : 'AFTER'.$lastrowid.'POS'.($i + 1)).'">';
+					print '<tr class="oddeven" id="row-'.(!empty($filearray[$key]['rowid'] && $filearray[$key]['rowid'] > 0) ? $filearray[$key]['rowid'] : 'AFTER'.$lastrowid.'POS'.($i + 1)).'">';
```
Comment (eldy) [inline htdocs/core/class/html.formfile.class.php:1303]: syntax error here.

Comment (grandoc) [reply htdocs/core/class/html.formfile.class.php]: Laurent
When I edit the file it's already corrected...

Comment (eldy) [reply htdocs/core/class/html.formfile.class.php]: Still some syntax errors. See travis report.

Comment (grandoc) [reply htdocs/core/class/html.formfile.class.php]: I know that it should be 👍 (!empty($filearray[$key]['rowid']**)** && $filearray[$key]['rowid'] > 0)
but when I edit the file it's already done...

**Pull request closed without merging**